backend/benefit/helsinkibenefit/settings.py

```diff
@@ -41,6 +41,10 @@
     CSRF_TRUSTED_ORIGINS=(list, []),
     YTJ_BASE_URL=(str, "http://avoindata.prh.fi/opendata/tr/v1"),
     YTJ_TIMEOUT=(int, 30),
+    ASSOCIATION_FORM_CODES=(
+        str,
+        "18,21,25,29,31,32,35,38,39,4,44,45,46,47,50,58,6,71,9,90,94999",
+    ),
```

Comment:
Adding e.g. a comment about what these numbers are or at least where they come from or perhaps a link to documentation from which they can be looked up from would add transparency to this magic number data.